crates/re_ui/src/lib.rs

```diff
@@ -88,3 +88,29 @@ pub fn apply_style_and_install_loaders(egui_ctx: &egui::Context) {
 
     design_tokens().apply(egui_ctx);
 }
+
+/// Used as a heuristic to figure out if it is safe to truncate text.
+///
+/// If this returns false, we should never truncate.
```

Comment:
> If this returns false, we should never truncate.

Double-negative. Would be easier to follow as "If this returns true, we should truncate text when necessary."

The rationale coupling this description to the name of the function still isn't totally obvious. Even if a panel is not resizable, if the text is much longer than the available (non-resizable) panel size, it seems like truncation could be necessary.

Reply:
I expanded the docstring to clarify